wikibase/queryService/api/Wikibase.js

```diff
@@ -79,21 +52,26 @@ wikibase.queryService.api.Wikibase = ( function( $ ) {
 	 * @return {jQuery.Promise}
 	 */
 	SELF.prototype.searchEntities = function( term, type, language ) {
-		var query = SEARCH_ENTITES;
-		query.search = term;
-
-		if ( type ) {
-			query.type = type;
-		}
-		if ( this._language || language ) {
-			query.language = language || this._language;
-			query.uselang = language || this._language;
-		} else {
-			query.language = LANGUAGE;
-			query.uselang = LANGUAGE;
-		}
-
-		return this._query( query );
+		var deferred = $.Deferred();
```

Comment:
this is europeana specific functionality, can we have a europeana.js file under the europenaa folder?